### DISCUSSION TITLE
cant open chatgpt Writer

### DISCUSSION THREAD

**Discussion by Omarbinhareb** [GorvGoyl/ChatGPT-Writer#72] [Report Bugs and Issues]
Why i cant open chatgpt Writer

**Comment by minchu04** [comment]
Sub: Submission of Original Medical Certificate for Medical reimbursement towards Chronic diseases in respect of family members of Ms. Nithyavathi B DM (HR):- Reg

Please find enclosed here with the Original Medical Certificate and Medical bills towards chronic diseases for medical reimbursement regarding required missed documents submitted by Smt.Nithyavathi B. DM (HR)

**Comment by Xavier035** [comment]
Por que eu não posso abrir chatgpt Writer
